themes-default/slim/static/js/templates/app-header.vue

```diff
@@ -112,14 +119,20 @@ module.exports = {
     data() {
         return {
             topMenuMapping: [
-                ['system', ['/home/restart', '/home/status', '/errorlogs', '/changes', '/news', '/IRC']],
                 ['home', ['/home', '/addShows', '/addRecommended']],
-                ['config', ['/config']],
-                ['history', ['/history']],
                 ['schedule', ['/schedule']],
+                ['history', ['/history']],
                 ['manage', ['/manage']],
+                ['config', ['/config']],
+                ['system', ['/home/restart', '/home/status', '/errorlogs', '/changes', '/news', '/IRC']],
```

Comment:
The order has meaning.
If you change the order, it will match the current path to an incorrect `topMenu`.
For example `/home/status` will match `home` instead of `system` with the change you made.